FieldOpt/Model/model.h

```diff
@@ -121,6 +125,22 @@ class Model : public Loggable
    private:
     Model *model_;
   };
+ public:
+  class economy{
+   public:
+    map<string, double> well_xy;
+    map<string, double> well_z;
+    map<string, double> well_lengths;
+    double n_wells = 0;
+    double costXY = 0;
+    double costZ = 0;
+    double cost = 0;
+    bool separate = false;
+    bool useWellCost = false;
+    QList<Wells::Well *> wells_;
+  };
+  economy wellCost(Settings::Optimizer::Objective);
```

Comment:
Public methods should have Capital First Letter (and yes, I know I haven't done a fantastic job keeping to this convention myself everywhere)